### DISCUSSION TITLE
No images in some feeds that used to have them not long ago

### DISCUSSION THREAD

**Discussion by s38b35M5** [spacecowboy/Feeder#231] [Q&A] [answered]
I am not sure which version this behavior began in, but I have a few feeds that used to have inline images in the native reader view that now show image placeholders for a time, and eventually resolve to an exclamation point. 

In some cases, the feed article list view shows thumbnails but the reader view does not. In others, there are no thumbnails loaded in the list view or reader view.

Feed examples with this problem: https://www.techdirt.com/feed/
https://pluralistic.net/feed/

Feed example without this problem: https://arstechnica.com/rss

Is this a problem with the app, the rss source, or something I can tweak on my end (without resorting to a browser view)?

**Comment by spacecowboy** [comment]
No idea. Techdirt seems to work fine for me.

The only way to know what's happening for YOU is to look at the logs (https://developer.android.com/tools/logcat)

**Reply by spacecowboy** [reply]
All logs of relevance can be seen with

```
adb logcat | grep FEEDER
```

**Reply by s38b35M5** [reply, marked as answer]
Thx for the nod in this direction. After your comment, I remembered my landlord said he put a new device between me and the internet a few weeks back. It turns out he was blocking all sorts of {notBad} URLs on a list he found somewhere. Things like *.wp.com and *.cloudfront.net .

I VPNd thru the device and everything loads fine. I'll ask him to remove the filter list. He likely thought it was a list of malware or ads he was blocking.

Thx again.

**Reply by s38b35M5** [reply]
@spacecowboy you're always very quick to respond and assist. I flipped a few coffees your way as a thank you. Thanks for this app.
